aries-site/src/examples/templates/dashboards/components/UpdateItem.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Anchor, Box, Button, Text } from 'grommet';
+import { FormDown, FormUp } from 'grommet-icons';
+
+export const UpdateItem = ({ title, description }) => {
+  // HPE Design System guidance for string length prior to truncation.
```

Comment:
I like the comment here. Is there a url we can include to allow the reader to see the guidance?

Also, we should qualify "string length prior to truncation" -- always? What about in a table cell? Basically we should say "... string length prior to truncation when ____ ____ ____"

Possible whens:
- Paragraph length (instead of string length)
- Presenting at-a-glance / summary / preview content in a card

Reply:
good thoughts I'll incorporate

Reply:
So I found that in the general section for `Card` there is guidance to limit to 100 characters when presenting content in a Card I added that in the comment 
https://design-system.hpe.design/components/card#general